addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -656,6 +672,23 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             this.resultsChart.setShowAnswers(showAnswers);
             this.resultsChart.updateChart();
         }
+    },
+    /**
+     * Updates the tooltip for current page (on right arrow icon for 'Next' content).
+     * this method will be called on Clicking of Next and Previous Arrow to show the
+     * tooltip for the Next Content.
+     */
+    _updateNextPageTooltip() {
+        let screenToDisplay = this._getNextScreen();
```

Comment:
Sugg:

```js
let tooltip;
if (this.currentScreen === 'startScreen') {
    tooltip = nextPageTooltips['startScreen'];
} else if (this.isLastQuestion && !this.isScoredQuestion && !this.sessionShowLeaderboard) {
    tooltip = nextPageTooltips['closingWords'];
} else {
    const nextScreen = this._getNextScreen();
    tooltip = nextPageTooltips[nextScreen];
}
```

That way we don't mix the next screen concept with the tooltip to display on the button.